.docker/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM ruby:2.6.2-alpine
+FROM ruby:2.6.5
```

@DarthHater this kind of raises an interesting question, in your opinion is alpine still serving us? 

@andrewrosario in lieu of @DarthHater weighing in, would you mind restoring the docker image to alpine? 

```suggestion
FROM ruby:2.6.5-alpine
```

---

It's buyers choice. All alpine gets the project is a smaller image to work with ultimately, but Alpine is as you know, tricky to work with. Since we don't use the image in production it's kinda whatever is best for developers. Slow download time vs lots of Alpine specific hacks.